.github/workflows/build-images.yml

```diff
@@ -21,24 +21,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        service: ["php-apache", "php-apache-nvm"]
         php: [7.3, 7.4, "8.0"]
```

Comment:
Adding a dimension to this matrix makes it so that `php-apache` and `php-apache-nvm` images are built for every version of `php` that we specify here.